### ISSUE TITLE
[iOS] Sharing to Keybase takes you to the app

### ISSUE THREAD

**Issue by YL-K** [keybase/client#23098]
Since the last update on iOS, Keybase opens itself when I'm trying to share links with people and then tries to show a dialogue (which breaks, if I tap it during the ~second it loads, requiring me to go back to the previous app and share with Keybase again), where it asks me if I want to save the link I'm trying to share as a file.

Please don't open Keybase when I'm trying to share a link with somebody. The previous UX was amazing and allowed super quick sharing. Now it's slow and tedious and a pain to use compared to any other e2e chat app.

**Comment by KizzyCode** [comment]
Yeah, the previous share workflow e.g. was this: Press share, select Keybase, select the recipient, send.

Now it’s for an image: Press share, select Keybase, rotate your phone (because Keybase has no support for landscape mode), flip the switch to send the full-size image, press "Chat", select the chat, send, go back to the previous app, rotate your phone again, recover the previous scrolling position. 

Honestly that’s pretty annoying and IMHO a huge regression compared to the previous sharing dialogue. The main problem is the app switch (wouldn’t surprise me if this is even HIG violation) – if you need more space for your controls, you could use a modal sheet like Signal does.
